pipeline_logs/firestore_pipeline_logs_table.py

```diff
@@ -0,0 +1,37 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials, firestore
+
+
+log = Logger(__name__)
+
+
+class FirestorePipelinesLogsTable(object):
```

Comment:
Consider FirestorePipelineLogger or just FirestoreLogger (pipeline could be considered implicit because this is in Pipeline-Infrastructure)

Reply:
updated to FirestorePipelineLogger